src/BloomExe/ImageProcessing/ImageServer.cs

```diff
@@ -79,7 +79,8 @@ protected override bool ProcessRequest(IRequestInfo info)
 			var r = GetLocalPathWithoutQuery(info);
 
 			// only process images
-			if(!IsImageTypeThatCanBeDegraded(r))
+			var isSvg = r.EndsWith(".svg", StringComparison.OrdinalIgnoreCase);
+			if (!IsImageTypeThatCanBeDegraded(r) && !isSvg)
```

Comment:
It's getting confusing whether the ImageServer is supposed to be responsible for all images or just ones that can be downgraded. Currently it's downgradable ones plus svgs...but that really just means all images we actually use. But it's main purpose is the downgrading. I've made https://trello.com/c/rQUlg1xo/24-improve-layering-of-file-server-classes for trying to sort this out. But this is better than just losing the SVGs.
I think we should wait for @hatton to comment, though. I pointed out the svg problem in the PR which removed handling them here, and he believed I had misunderstood what was needed, so we may still need to go some other direction.